packages/react-native-editor/.eslintrc.js

```diff
@@ -16,16 +16,11 @@ module.exports = {
 		"react",
 		"react-native",
 		"jest",
-		"flowtype"
```

Comment:
I haven't converted flow files to esnext yet but I just wanted to have less dependencies to deal with